hedera-node/hedera-app/src/main/java/com/hedera/node/app/MonoServicesMain.java

```diff
@@ -98,6 +100,12 @@ private void initApp() {
     }
 
     private void doStagedInit() {
+        // Construct BootstrapProperties with false to suppress redundant property listing in logs
+        final var disableJULogging = new BootstrapProperties(false).getBooleanProperty(NETTY_LOGGING_OFF);
+        if (disableJULogging) {
+            java.util.logging.LogManager.getLogManager().reset();
```

Comment:
This does not only deactive specific logging config for netty but for all parts that use JuL. Instead of doing so we should define / overwrite specific rules for netty.